packages/introspect/src/generate.ts

```diff
@@ -244,12 +276,21 @@ function fullTableName(table: TableDetails, defaultSchema: string) {
 function createTableDefs(
     tables: TableDetails[],
     defaultSchema: string,
+    schemaDefOverrides: SchemaDefOverrides | undefined,
 ): Record<string, JSONSchema> {
     const empty: Record<string, JSONSchema> = {};
 
     return tables.reduce((acc, table) => {
-        acc[createTableDefId(table, defaultSchema)] =
-            createTableJsonSchema(table);
+        const tableSchemaOverrides = schemaDefOverrides
```

Comment:
you don't need a call to `.includes`. This should be a simple call to `schemaDefOverride[table]`